sync_ci/Makefile

```diff
@@ -31,6 +31,7 @@ install:
 	@make binary
 	@mv bin/sync-ci-service $(INSTALL_DIR)/sync-ci-service
 	@cp pkg/parser/envrules.json $(INSTALL_DIR)/envrules.json
+	@cp pkg/parser/regex_rules.json (INSTALL_DIR)/regex_rules.json
```

Comment:
add `$` before INSTALL_DIR

Reply:
add `WorkingDirectory=$(INSTALL_DIR)` under `[Service]` to make the app find the regex_rules.json path

Reply:
Got